conanfile.py

```diff
@@ -1,13 +1,15 @@
 # pylint: disable=no-member
 
-import os
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
```

Comment:
Is it normal to use `from` in conanfiles? It is akin to `using` in C++, which is frowned upon.

Reply:
Yes, look at example https://github.com/conan-io/conan-center-index/blob/master/recipes/abseil/all/conanfile.py#L1